validator.go

```diff
@@ -87,7 +90,7 @@ func (v *validator) Validate(claims Claims) error {
 
 	// We always need to check the expiration time, but usage of the claim
```

Comment:
small nit: The documentation should be changed here, that it is optional by default (in accordance to the RFC) but can be changed with the requireExp variable.

Reply:
done